tests/Makefile

```diff
@@ -22,9 +22,9 @@ external-backup-tests:
 	@GOTAGS=$(GOTAGS),test,externalbackups ../scripts/go-test.sh -cover -v -run TestGCSExternalBackup 2>&1 | tee test.log
 	@$(MAKE) report JUNIT_OUT=external-backup-tests-results
 
-.PHONY: compliance-tests
-compliance-tests:
-	../scripts/go-test.sh -cover -v -run compliance_operator_test.go 2>&1 | tee test.log
-	@$(MAKE) report JUNIT_OUT=compliance-tests-results
+.PHONY: compliance-v2-tests
+compliance-v2-tests:
```

Comment:
Was this target only used in this one path?  I ask because if it was used elsewhere in tests that actually worked, we should keep that original target and make another one.  If this is the only path that uses it then this is OK since that target didn't work in this path.

Reply:
Yep - I believe the only path that used this target was the `run-compliance-e2e.sh` script:

https://github.com/stackrox/stackrox/commit/f0a51597676fdde7e5bd638b215e154af16bb24c